bigchaindb/tendermint/lib.py

```diff
@@ -23,23 +23,29 @@
 
 class BigchainDB(Bigchain):
 
-    def post_transaction(self, transaction):
+    def post_transaction(self, transaction, mode):
         """Submit a valid transaction to the mempool."""
+        mode_list = ('broadcast_tx_async',
+                     'broadcast_tx_sync',
+                     'broadcast_tx_commit')
+        if not mode or mode['mode'] not in mode_list:
```

Comment:
I think adding parantheses around the appropriate sub-conditions would add more clarity.

Reply:
I decided to not add them since it is still readable. But I'm also still open to change it if more people think we should have parentheses in this case.